xero-payroll-au.yaml

```diff
@@ -875,6 +1189,252 @@ paths:
+          ruby: XeroRuby::PayrollAu::EarningsType::OVERTIMEEARNINGS
+          python_string: OVERTIMEEARNINGS
+          java: com.xero.models.payrollau.EarningsType.OVERTIMEEARNINGS
+          csharp: EarningsType.OVERTIMEEARNINGS
+          object: earningsRate
+          is_last: true 
+        - deductionType:
+          key: deductionType
+          keyPascal: DeductionType
+          keySnake: deduction_type
+          is_object: true 
+        - deductionTypeName:
+          key: name
+          keyPascal: Name
+          keySnake: name
+          default: Union Fees
+          object: deductionType
+        - deductionAccountCode:
+          key: accountCode
+          keyPascal: AccountCode
+          keySnake: account_code
+          object: deductionType
+          default: 826
+        - reducesTax:
+          key: reducesTax
+          keyPascal: ReducesTax
+          keySnake: reduces_tax
+          object: deductionType
+          default: true
+          nonString: true
+        - reducesSuper:
+          key: reducesSuper
+          keyPascal: ReducesSuper
+          keySnake: reduces_super
+          object: deductionType
+          default: true
+          nonString: true
+          is_last: true 
+        - leaveType:
+          key: leaveType
+          keyPascal: LeaveType
+          keySnake: leave_type
+          is_object: true 
+        - leaveTypeName:
+          key: name
+          keyPascal: Name
+          keySnake: name
+          default: Annual Leave
+          object: leaveType
+        - leaveTypeOfUnits:
+          key: typeOfUnits
+          keyPascal: TypeOfUnits
+          keySnake: type_of_units
+          object: leaveType
+          default: Hours
+        - isPaidLeave:
+          key: isPaidLeave
+          keyPascal: IsPaidLeave
+          keySnake: is_paid_leave
+          object: leaveType
+          default: true
+          nonString: true
+        - showOnPayslip:
+          key: showOnPayslip
+          keyPascal: ShowOnPayslip
+          keySnake: show_on_payslip
+          object: leaveType
+          default: true
+          is_last: true 
+          nonString: true
+        - reimbursementType:
+          key: reimbursementType
+          keyPascal: ReimbursementType
+          keySnake: reimbursement_type
+          is_object: true 
+        - reimbursementTypeName:
+          key: name
+          keyPascal: Name
+          keySnake: name
+          default: Annual Leave
+          object: reimbursementType
+        - reimbursementTypeAccountCode:
+          key: accountCode
+          keyPascal: AccountCode
+          keySnake: account_code
+          object: reimbursementType
+          is_last: true 
+          default: 850            
+        - earningsRates:
+          is_list: true 
+          key: earningsRates
+          keyPascal: EarningsRates
+          keySnake: earnings_rates
+          csharp: EarningsRate
+          java: EarningsRate
+        - add_earningsRates:
```

Comment:
should these param names be half snake? Don't think i've seen that anywhere else 🤔

Reply:
![](https://media.giphy.com/media/q8C0Ljmy4F6Ss/giphy.gif)

Reply:
@SerKnight 

![image](https://user-images.githubusercontent.com/79074721/118415234-9e577180-b6ec-11eb-8401-ddaa74be800b.png)

I actually found 88 occurences in xero_accounting.yaml
Did we just invent half snake case?

Looks like the naming convention for those params seems to be camelCase in general so I'll go ahead and change those in payroll-au and accounting